rviz_default_plugins/CMakeLists.txt

```diff
@@ -599,6 +609,7 @@ if(BUILD_TESTING)
   if(TARGET ros_image_texture_test)
     target_include_directories(ros_image_texture_test PUBLIC ${TEST_INCLUDE_DIRS})
     target_link_libraries(ros_image_texture_test ${TEST_LINK_LIBRARIES})
+    ament_target_dependencies(ros_image_texture_test ${TEST_TARGET_DEPENDENCIES})
```

Comment:
```
[ 42%] Linking CXX executable ros_image_texture_test
/usr/bin/ld: CMakeFiles/ros_image_texture_test.dir/test/rviz_default_plugins/displays/image/ros_image_texture_test.cpp.o: undefined reference to symbol '_ZN4Ogre10LogManager9createLogERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEEbbb'
/usr/bin/ld: /home/jenkins-agent/workspace/ci_linux/ws/install/rviz_ogre_vendor/opt/rviz_ogre_vendor/lib/libOgreMain.so.1.12.1: error adding symbols: DSO missing from command line
collect2: error: ld returned 1 exit status
make[2]: *** [CMakeFiles/ros_image_texture_test.dir/build.make:242: ros_image_texture_test] Error 1
make[1]: *** [CMakeFiles/Makefile2:321: CMakeFiles/ros_image_texture_test.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs..
```

```console
$ c++filt _ZN4Ogre10LogManager9createLogERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEEbbb
Ogre::LogManager::createLog(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, bool, bool, bool)
```

SO post about `DSO missing from command line: https://stackoverflow.com/questions/19901934/libpthread-so-0-error-adding-symbols-dso-missing-from-command-line

No idea how to fix. Maybe `TEST_TARGET_DEPENDENCIES` needs the package `rviz_ogre_vendor` or `rviz_common`? Maybe `ament_target_dependencies(rviz_default_plugins ...` needs `PUBLIC`?